<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Designer::Internal::NewClassWidget</class>
 <widget class="QWidget" name="Designer::Internal::NewClassWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>194</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="classNameLabel">
     <property name="text">
      <string>&amp;Class name:</string>
     </property>
     <property name="buddy">
      <cstring>classLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Utils::ClassNameValidatingLineEdit" name="classLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="baseClassLabel">
     <property name="text">
      <string>&amp;Base class:</string>
     </property>
     <property name="buddy">
      <cstring>baseClassComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="baseClassComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="classTypeLabel">
     <property name="text">
      <string>&amp;Type information:</string>
     </property>
     <property name="buddy">
      <cstring>classTypeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="classTypeComboBox">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inherits QObject</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inherits QWidget</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inherits QDeclarativeItem - Qt Quick 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inherits QQuickItem - Qt Quick 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Based on QSharedData</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&amp;Header file:</string>
     </property>
     <property name="buddy">
      <cstring>headerFileLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="Utils::FileNameValidatingLineEdit" name="headerFileLineEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="sourceLabel">
     <property name="text">
      <string>&amp;Source file:</string>
     </property>
     <property name="buddy">
      <cstring>sourceFileLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="Utils::FileNameValidatingLineEdit" name="sourceFileLineEdit"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="formLabel">
     <property name="text">
      <string>&amp;Form file:</string>
     </property>
     <property name="buddy">
      <cstring>formFileLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="Utils::FileNameValidatingLineEdit" name="formFileLineEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="pathLabel">
     <property name="text">
      <string>&amp;Path:</string>
     </property>
     <property name="buddy">
      <cstring>pathLabel</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="Utils::PathChooser" name="pathChooser" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::ClassNameValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/classnamevalidatinglineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::FileNameValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/filenamevalidatinglineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
